frontend/src/Judge/components/OtherMediaSubmission.js

```diff
@@ -1,5 +1,111 @@
-import React from 'react'
+import React, { Component, Fragment } from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import Lightbox from 'react-images'
+import { Alert } from 'reactstrap'
 
-const OtherMediaSubmission = props => <div>Other Submission</div>
+import pdf from 'assets/pdf.svg'
+
+const PhotoContainer = styled.img`
+  height: 90%;
+  object-fit: scale-down;
```

Comment:
`object-fit` is not supported in IE 11 which we need to support because of Windows 7: https://caniuse.com/#feat=object-fit